langserver/symbol.go

```diff
@@ -27,31 +27,37 @@ import (
 type query struct {
 	kind   lsp.SymbolKind
 	filter filterType
+	file   string
 	tokens []string
 }
 
 // parseQuery parses a user's raw query string and returns a
-// structured representation of the query.
-func parseQuery(q string) query {
-	var qu query
-	toks := tokenizer.Split(strings.ToLower(q), -1)
-	index := 0
-	for _, tok := range toks {
-		if tok == "" {
+// structured representation of the query. Possible queries are:
+func parseQuery(q string) (qu query) {
```

Comment:
Per discussion on Slack, will leave it as-is. I also agree it's better left in the func signature when the func is unexported.